healpy/projaxes.py

```diff
@@ -915,7 +891,7 @@ def create_colormap(cmap, badcolor="gray", bgcolor="white"):
             "newcm", cmap0._segmentdata, cmap0.N
         )
     else:
-        newcm = copy.deepcopy(cmap0)
+        newcm = copy.copy(cmap0)
```

Comment:
@kadrlica compared to https://github.com/healpy/healpy/pull/627, I was getting a warning with `deepcopy` and not with `copy`

Reply:
So this should probably go in a separate PR.

Reply:
right, moved to https://github.com/healpy/healpy/pull/681